### PR TITLE
Bump version to 3.12.0

### PR DESCRIPTION
**Pull request by DFolchA**
Hi team,

This PR adds the v3.12.0 specs files.

Regards.

### CODE REVIEW

File: debs/SPECS/3.12.0/wazuh-api/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 7 Oct 2019 13:33:00 +0000
+
+wazuh-api (3.10.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Tue, 24 Sep 2019 10:19:00 +0000
```
Comment (okynos) [inline debs/SPECS/3.12.0/wazuh-api/debian/changelog]: Remove `3.10.3` version from changelog please.

File: debs/SPECS/3.12.0/wazuh-api/debian/control [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Section: admin
+Priority: extra
+Maintainer: Wazuh, Inc <info@wazuh.com>
+Build-Depends: debhelper (>= 7.0.50~), nodejs (>= 4.6), npm, curl, gawk
+Standards-Version: 3.8.4
+Homepage: http://www.wazuh.com
+
+Package: wazuh-api
+Architecture: any
+Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.11.0), wazuh-manager (<< 3.11.1), curl
```
Comment (okynos) [inline debs/SPECS/3.12.0/wazuh-api/debian/control]: Please change `3.11` to `3.12` on `wazuh-api` control file.

File: debs/SPECS/3.12.0/wazuh-manager/debian/changelog [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 7 Oct 2019 13:33:00 +0000
+
+wazuh-manager (3.10.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Tue, 24 Sep 2019 10:19:00 +0000
```
Comment (okynos) [inline debs/SPECS/3.12.0/wazuh-manager/debian/changelog]: Remove `3.10.3` version from changelog please.

File: rpms/SPECS/3.12.0/wazuh-agent-3.12.0.spec [hunk 4]
```diff
@@ -0,0 +1,754 @@
...
+%attr(640,root,ossec) %{_localstatedir}/ossec/wodles/oscap/content/*
+
+
+%changelog
+* Mon Oct 18 2019 support <info@wazuh.com> - 3.12.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Mon Oct 7 2019 support <info@wazuh.com> - 3.11.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Tue Sep 24 2019 support <info@wazuh.com> - 3.10.3
+- More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (okynos) [inline rpms/SPECS/3.12.0/wazuh-agent-3.12.0.spec]: Remove `3.10.3` version from changelog please.

File: rpms/SPECS/3.12.0/wazuh-api-3.12.0.spec [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/logs/api
+%attr(660, ossec, ossec) %ghost %{_localstatedir}/ossec/logs/api.log
+
+%changelog
+* Mon Oct 18 2019 support <info@wazuh.com> - 3.12.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Mon Oct 7 2019 support <info@wazuh.com> - 3.11.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Tue Sep 24 2019 support <info@wazuh.com> - 3.10.3
+- More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (okynos) [inline rpms/SPECS/3.12.0/wazuh-api-3.12.0.spec]: Remove `3.10.3` version from changelog please.

File: rpms/SPECS/3.12.0/wazuh-manager-3.12.0.spec [hunk 6]
```diff
@@ -0,0 +1,1022 @@
...
+
+%{_initrddir}/*
+
+%changelog
+* Mon Oct 18 2019 support <info@wazuh.com> - 3.12.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Mon Oct 7 2019 support <info@wazuh.com> - 3.11.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Tue Sep 24 2019 support <info@wazuh.com> - 3.10.3
+- More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (okynos) [inline rpms/SPECS/3.12.0/wazuh-manager-3.12.0.spec]: Remove `3.10.3` version from changelog please.

File: solaris/solaris11/SPECS/template_agent_v3.12.0.json [hunk 7]
```diff
@@ -0,0 +1,1635 @@
+{
```
Comment (okynos) [inline solaris/solaris11/SPECS/template_agent_v3.12.0.json]: Check if this template is right, please.

File: debs/SPECS/3.12.0/wazuh-agent/debian/changelog [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 7 Oct 2019 13:33:00 +0000
+
+wazuh-agent (3.10.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Tue, 24 Sep 2019 10:19:00 +0000
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply debs/SPECS/3.12.0/wazuh-agent/debian/changelog]: Done in https://github.com/wazuh/wazuh-packages/pull/338/commits/7f378b0e2a608623794c8b1a06e0d78beaf4777d

File: debs/SPECS/3.12.0/wazuh-api/debian/changelog (hunk 1 above)
Comment (DFolchA) [reply debs/SPECS/3.12.0/wazuh-api/debian/changelog]: Done in 7f378b0

File: debs/SPECS/3.12.0/wazuh-api/debian/control (hunk 2 above)
Comment (DFolchA) [reply debs/SPECS/3.12.0/wazuh-api/debian/control]: Done in 7f378b0

File: debs/SPECS/3.12.0/wazuh-manager/debian/changelog (hunk 3 above)
Comment (DFolchA) [reply debs/SPECS/3.12.0/wazuh-manager/debian/changelog]: Done in 7f378b0

File: rpms/SPECS/3.12.0/wazuh-agent-3.12.0.spec (hunk 4 above)
Comment (DFolchA) [reply rpms/SPECS/3.12.0/wazuh-agent-3.12.0.spec]: Done in 7f378b0

File: rpms/SPECS/3.12.0/wazuh-api-3.12.0.spec (hunk 5 above)
Comment (DFolchA) [reply rpms/SPECS/3.12.0/wazuh-api-3.12.0.spec]: Done in 7f378b0

File: rpms/SPECS/3.12.0/wazuh-manager-3.12.0.spec (hunk 6 above)
Comment (DFolchA) [reply rpms/SPECS/3.12.0/wazuh-manager-3.12.0.spec]: Done in 7f378b0

File: solaris/solaris11/SPECS/template_agent_v3.12.0.json (hunk 7 above)
Comment (DFolchA) [reply solaris/solaris11/SPECS/template_agent_v3.12.0.json]: Done in 7f378b0

**Comment by alberpilot** [comment]
Discarded OR. Sorry for the inconvenience.

**Pull request closed without merging**